### ISSUE TITLE
Bug with adding a new item to layout

### ISSUE THREAD

**Issue by alanrubin** [STRML/react-grid-layout#19]
I found a strange bug when adding a new item to layout. The new item is being positioned at a strange position, outside the layout. If I drag other items after the new is added, suddenly the new item goes to the right position.

After a brief investigation, I found out that for the new item, along with the standard CSS -xx-transform: translate(...) styles being added, a "top" and "left" are being added so that adds to the transform putting the new item in wrong position. You can see this behaviour happening in the example "dynamic add/remove" page as well.

This seems to be caused by the GridItem.js "mixinDraggable" function - there the useCSSTransforms property is false at first because this.isMounted() returns false. When some dragging occur, then it becames true as the element is already mounted.

So by changing the line inside function "mixinDraggable"
```
useCSSTransforms: this.props.useCSSTransforms && this.isMounted()
```
to
```
useCSSTransforms: this.props.useCSSTransforms
```
seems to fix the issue. 

I was happy to submit a PR but I was wondering if not checking if the element isMounted could cause a regression somewhere else. What do you think ?

**Comment by jardakotesovec** [comment]
Same issue here. I can confirm success with @alanrubin fix, but I am also not competent to say if its correct place to fix this.

**Comment by STRML** [comment]
This was a fix for a server rendering checksum issue, but I don't believe it works 100%. I'm going to take a few stabs at it and I'll get back to you guys.

**Comment by STRML** [comment]
This is fixed in `0.7.0`, thanks guys.
